### PR TITLE
initial commit for eventcard converter

### PR DESCRIPTION
**Pull request by sharanagoudapatil**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://70-create-events-card-converter--danaher-ls-aem--hlxsites.hlx.page/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "version": "1.2.2",
   "description": "Starter project for Adobe Helix",
   "scripts": {
-    "lint:js": "eslint .",
+    "lint:js": "eslint . --fix",
```
Comment (bpauli) [inline package.json:7]: I think we should not apply fixes automatically.

File: tools/importer/import.js [hunk 2]
```diff
@@ -135,6 +135,63 @@ const createCards = (main, document) => {
...
+        eventDateDiv.append(a);
+
+        cards.push([eventDateDiv]);
+      }
+    });
+    //const cells = [[`Cards (eventcard, columns ${numCols})`], ...cards];
+    const cells = [['Cards (eventcard)'], ...cards];
+
+    if (cards.length > 0) {
+      fl.before(document.createElement('hr'));
```
Comment (bpauli) [inline tools/importer/import.js:188]: We should only create a new section when this is really required (for example by provided a dedicated section metadata)

**Review by bpauli — CHANGES_REQUESTED** [review]
@sharanagoudapatil The PR looks already very good. I added my suggested improvements. Beside that,  it would be great if you can modify the existing `createFullLayoutSection` so we only add a section metadata to a block when the styles are not empty. For the Landingpage we didn't have this case.

**Review by bpauli — APPROVED** [review]

**Comment by sharanagoudapatil** [comment]
@bpauli - Thanks much for your time/help to connect on this morning. 
Now we are good with the changes. Thanks.

**Pull request merged**